ts/src/bybit.ts

```diff
@@ -1497,6 +1497,7 @@ export default class bybit extends Exchange {
     }
 
     async fetchDerivativesMarkets (params) {
+        params = this.extend (params);
```

Comment:
Please combine this line with the line below.

Reply:
do you mean `params = this.extend (params, {x:y});` ?
I have thought about it, but creating additional object is not better than just assigning a value to it's key. should I still do

Reply:
@ttodua actually, you may be right, in this case assigning a value under the assoc key is probably faster than creating a new object… let’s keep it this way then.